src/watchlists/EditButton.jsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import { MaterialIcons } from '@expo/vector-icons';
-import { View, TouchableHighlight } from 'react-native';
+import { View, Text,TouchableHighlight } from 'react-native';
 import PropTypes from 'prop-types';
 import styles from './styles';
 
-const EditButton = ({ isInEditMode, toggleEditMode }) => (
+const EditButton = ({ isInEditMode, toggleEditMode, isEmptyWatchList }) => (
   <View style={styles.editor}>
+    <View style={styles.editorText}>{isEmptyWatchList ? <Text style={{ color: '#ccc' }}>This WatchList is Empty</Text> : null}</View>
```

Comment:
this should be in the watchlist component, not in the `EditButton` component, they are two different things. What will we do if we add more buttons to the footer or we just remove the edit button

Reply:
it can go between the watchlist title and edit button, it is not necessary tu pot them in the same line